airflow/example_dags/example_python_operator.py

```diff
@@ -34,7 +34,7 @@
     # [START howto_operator_python]
     def print_context(ds, **kwargs):
         """Print the Airflow context and ds variable from the context."""
-        pprint(kwargs)
+        print(kwargs)
```

Comment:
`pprint` is correct, check the import line on L22

https://github.com/apache/airflow/blob/463ed82255403cd5663160140c44987d61e893a3/airflow/example_dags/example_python_operator.py#L22

Reply:
oh, sorry for bothering, shame on me, too inattentive late at night :/